packages/manager/modules/pci/src/projects/projects.constant.js

```diff
@@ -1,7 +1,37 @@
 export const DEFAULT_PROJECT_KEY = 'PUBLIC_CLOUD_DEFAULT_PROJECT';
 export const QUOTA_THRESHOLD = 80;
 
+export const ORDER_FOLLOW_UP_POLLING_INTERVAL = 7000;
+
+export const ORDER_FOLLOW_UP_STATUS_ENUM = {
+  DOING: 'DOING',
+  DONE: 'DONE',
+  ERROR: 'ERROR',
+  TODO: 'TODO',
+};
+
+export const ORDER_FOLLOW_UP_STEP_ENUM = {
+  AVAILABLE: 'AVAILABLE',
+  DELIVERING: 'DELIVERING',
+  VALIDATED: 'VALIDATED',
+  VALIDATING: 'VALIDATING',
+};
+
+export const ORDER_FOLLOW_UP_STEP_VALIDATING_HISTORY = {
+  PREFIX: 'FRAUD_',
+  ENUM: {
```

Comment:
Enums aren't directly exposed by the API?
I'm asking that in order to avoid to maintain a list in a constant files.

Reply:
No, it's note provided.

Reply:
Look like `billing.order.followUp.HistoryStatusEnum` contains the values in `me.json` schema.

Reply:
fixed